compiler/rustc_lexer/src/cursor.rs

```diff
@@ -81,4 +81,13 @@ impl<'a> Cursor<'a> {
 
         Some(c)
     }
+
+    /// Eats symbols while predicate returns true or until the end of file is reached.
+    pub(crate) fn eat_while(&mut self, mut predicate: impl FnMut(char) -> bool) {
+        // I tried making optimized version of this for eg. line comments, but
```

Comment:
A first-person pronoun looks very odd here.

Reply:
Yeah, but I don't know what would be better. "we" doesn't look better either.

Reply:
I suggest dropping the comment. It looks more like a comment you'd say here on GitHub.

Reply:
No opinion here.

Reply:
Rephrased it to "It was tried". 
I'd like to leave that comment though, because it's the result of a day of playing around with ways to do poor man's SIMD to optimize whitespace and line comment lexing. That didn't work out, but I discovered the improvement in this PR.